_lessons/18-adding-some-color.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Adding a Splash of Color
+lesson: 18
+layout: default
+---
+
+It's time to spruce up our page with a splash of color! In this section, we'll learn how to use the browser inspector to preview colors on our page in real time. The browser inspector is an incredible development tool available in most browsers. 
+
+**Note on Browsers** Chrome and Firefox have the best developer tools, so I recommend using one of those browsers. They are both free to download. However, the Inspector is also available in Edge and Safari. In Safari, you may need to enable it in the settings.
+
+### Choosing a Background with the Inspector
+
+First, let's add a background color to the `body`, to change the background of the entire page. I will set it to white for now (`#ffffff`) - we will change this in a minute. 
+
+```
+body {
+  background-color: #ffffff; 
+}
+```
+
+Since this affects the `body`, place this rule at the top of your CSS. Generally, the more of the page something affects, you'll want to put it near the top so it's easier to find and see. 
+
+Now, open the file in your browser, or refresh the page if you already had it open. Right-click anywhere on the background of the page, and select "Inspect". (In some browsers, the option may be called "Inspect Element" instead.)
+
+This will open up a new panel which might open on the bottom or side of your browser. This panel is called the Inspector, and comes with several tools to poke around the code for your page!
+
+The Inspector has a few different sections. In one section, you'll see the HTML code of your page, with an element highlighted. If it's not already highlighted, click on the `<body>` tag to highlight the body element. 
+
+<div style="text-align: center">
+  <img src="/assets/ch18_1.png" />
+</div>
+
+**Note** These screenshots were taken in Firefox, but other browsers have the same features; it will just look slightly different. I am also using dark mode; your inspector might be white instead. 
+
+Now, either under it or to the side, you'll see a separate section that has all the CSS code that applies to the `body`. It will look something like this: 
+
+<div style="text-align: center">
+  <img src="/assets/ch18_2.png" />
+</div>
+
+**Note** You might see some CSS here that you didn't write. I have some in my screenshot. This code comes from browser extensions and can be safely ignored. 
```

Comment:
Can you temporarily disable your extensions for the purpose of this screenshot? This is going to be a bunch of confusing noise to beginners.